tests/test_cbv.py

```diff
@@ -56,12 +79,39 @@ def __init__(self, two: int = Depends(dependency_two)):
             def int_dependencies(self) -> int:
                 return self.one + self.two
 
-        client = TestClient(router)
+        return
+
+    def test_dependencies(self, cbv_router: None, client: TestClient) -> None:
         response = client.get("/")
-        assert response.status_code == 200
-        assert response.content == b"3"
+        assert response.status_code == SUCCESS_CODE
+        assert response.content == THREE_AS_BYTES

```

Comment:
`assert response.json() == 3` maybe?

Reply:
also does not work - it returns b'3' and decoding it retuns "3" as a string- it is possible but we will need to do some manipulation on the respose - and I  am not sure it is favorable as we want to check the response as is